var/spack/repos/builtin/packages/gnutls/package.py

```diff
@@ -39,4 +39,8 @@ class Gnutls(AutotoolsPackage):
 
     version('3.3.9', 'ff61b77e39d09f1140ab5a9cf52c58b6')
 
-    depends_on("nettle")
+    # configure sez: Note that this version of gnutls doesn't support 
+    # nettle 3.0.
+    depends_on("nettle@:2.7.1")
```

Comment:
Will it work with nettle 2.8 or 2.9? What exactly does configure say? We don't want to have to update this dependency every time a new brain of nettle comes out.